lib/screens/list/widgets/generic_card.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import '../../../constants/colors.dart';
+import '../../home/widgets/menu_avatar.dart';
+
+class GenericCard extends StatelessWidget {
+  const GenericCard({
+    super.key,
+    required this.name,
+    this.borderColor,
+    this.img,
+    this.url,
+    this.outlineImg,
+    this.onTap,
+  });
+
+  final Border? borderColor;
+  final String name;
+  final String? img;
+  final String? url;
+  final bool? outlineImg;
+  final VoidCallback? onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    final showImg = img != null || url != null;
+    return GestureDetector(
+      onTap: () => onTap?.call(),
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
```

Comment:
Please define a `final or a const` for these values at the beginning of the function to reduce random values spread. For these cases we could have a `Spacing` class with values to reuse among the app